Remotion/ObjectBinding/Validation.UnitTests/BusinessObjectValidationResultTest.cs

```diff
@@ -153,7 +154,7 @@ public void GetValidationFailures_ReturnsMatchingValidationFailures ()
       Assert.That(validationResult.IsValid, Is.False);
       Assert.That(validationResult.Errors.Count, Is.EqualTo(4));
       Assert.That(
-          validationResult.Errors.OfType<PropertyValidationFailure>().Select(f => f.ValidatedProperty).Distinct().Count(),
+          validationResult.Errors.Select(f => f.ValidatedProperties.Select(v => v.Property)).Distinct().Count(),
```

Comment:
This rewrite is incorrect. We need to use SelectMany, otherwise we validate that there's a Distinct objects (here: instances of IEnumerable) instead of disinct properties.
` validationResult.Errors.SelectMany(f => f.ValidatedProperties.Select(vp => vp.Property)).Distinct().Count(),`

The name of the expression variable: typically choses is the (most expressive) noun for the single character or multiple if it's practical. "ValidatedProperty" uses "Property" as the sole noun, so it's "p" or "vp". Since we the write "property.Property" it's a bit clearer to use "vp". I'm fine with either, though.